src/olympia/files/migrations/0006_auto_20210223_1215.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.18 on 2021-02-23 12:15
+
+from django.db import migrations
+
+from olympia.constants.platforms import PLATFORM_ALL
+
+
+def set_all_files_platform_to_all(apps, schema_editor):
+    File = apps.get_model('files', 'File')
+    # At this point we should no longer have more than one File per Version.
+    # We already set platform=ALL on Files on Versions that did have duplicate
+    # Files, and we've been setting it for new Files for a while now as well,
+    # now we need to do it for all the rest of them.
+    File.objects.exclude(platform=PLATFORM_ALL.id).update(platform=PLATFORM_ALL.id)
```

Comment:
to confirm, there's no chance of this failing because there are some rogue versions that still have multiple files in dev or stage? (even deleted addons/versions)

Reply:
No there shouldn't be any duplicates left anywhere, and I don't think there was a unique constraint anyway.